contracts/modules/SSVClusters.sol

```diff
@@ -344,4 +332,10 @@ contract SSVClusters is ISSVClusters {
 
         emit ClusterWithdrawn(msg.sender, operatorIds, amount, cluster);
     }
+
+    function exitValidator(bytes calldata publicKey, uint64[] calldata operatorIds) external override {
+        ValidatorLib.validateState(publicKey, operatorIds, SSVStorage.load());
+
+        emit ValidatorExited(publicKey, operatorIds);
+    }
```

Comment:
@lior-blox At what point do we minus the validator from the operator list?  If we dont, cant it be a hack that people can fill an operator with 500 validators then exit them and the operator is then trapped?

If its not dkg then how are you going to generate a keystore to perform the exit without a mnemonic?  Also how will it know the validators index (not beacon index but validator index from the mnemonic)

Also what if someone does this contract call to exit then does resharing with new operators with dkg right after this call?